src/e2e-test/java/io/cdap/plugin/servicenow/stepsdesign/DesignTimeSteps.java

```diff
@@ -72,23 +68,26 @@ public void clickOnAddConnectionButton () {
 
   @Then("Click on the ServiceNow Connector button")
   public void clickOnServiceNowConnectorButton () {
-    ServiceNowPropertiesPageActions.clickServiceNowConnectorButton();}
+    ServiceNowPropertiesPageActions.clickServiceNowConnectorButton();
+  }
 
   @Then("Click on the Test Connection button")
   public void clickOnTestConnectionButton () {
-    ServiceNowPropertiesPageActions.clickTestConnectorButton();}
+    ServiceNowPropertiesPageActions.clickTestConnectorButton();
+  }
 
   @Then("Click on the Create button")
   public void clickOnCreateButton () {
-    ServiceNowPropertiesPageActions.clickCreateButton();}
-
-  @Then("Click on the Servicenowconnection Button")
-  public void clickOnServicenowConnectionButton () {
-    ServiceNowPropertiesPageActions.clickServicenowConnectionButton();}
-
+    ServiceNowPropertiesPageActions.clickCreateButton();
+  }
+  @Then("Click on the ServiceNowConnection Button")
```

Comment:
Change ServiceNowConnection to Connection

Reply:
DONE